### PR TITLE
pull dob from person case for child health case

### PR DESCRIPTION
**Pull request by Rohit25negi**
This fixes this: https://app.asana.com/0/1118273917150202/1125465164253964/f

Go throw commit by commit for easy review

### CODE REVIEW

File: custom/icds_reports/tests/fixtures/child_cases.csv [hunk 1]
```diff
@@ -1299,8 +1299,8 @@ ff7feb43-b8d7-4ad0-939c-553e36acf8b0,2017-07-01,Name 2381,2017-06-28,2017-04-19,
...
-test_child_case_3,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_3,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a103,s22,b7,d6,st5,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_4,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_4,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a104,s23,b8,d7,st6,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
-test_child_case_5,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_5,ab2ab93e-d927-421f-9d39-b8183ab96aac,1d05300f-ff4a-4151-adae-f117e84dfdbf,a105,s24,b9,d8,st7,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
+test_child_case_1,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_1,ab2ab93e-d927-421f-9d39-b8183ab96aac,new_person_case1,a101,s20,b5,d4,st3,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
```
Comment (emord) [inline custom/icds_reports/tests/fixtures/child_cases.csv]: By "correct the fixture", did you mean that these child_health cases were originally using the same person case (which is an invalid state)?

In the future can you add that information in a github comment, or the commit body?

File: custom/icds_reports/tests/fixtures/person_cases.csv [hunk 2]
```diff
@@ -1,2107 +1,3059 @@
...
-223011de-ecf3-4b5e-beca-dec2fe0d2f83,2017-07-01 00:00:00,a50,s2,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-05-02,,,,
-2360319e-67f0-44bd-8fa2-d25a79bbe225,2017-07-01 00:00:00,a33,s1,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-23df8b5b-47d2-4a3b-9b77-583a45550438,2017-07-01 00:00:00,a33,s1,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-2494d2b9-088e-4f6b-aa60-fa160b3949f7,2017-07-01 00:00:00,a18,s2,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-29fedcb2-7487-47fc-a0ed-878102944aad,2017-07-01 00:00:00,a43,s3,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-2a25269b-b164-48a4-9620-a89509cb6343,2017-07-01 00:00:00,a18,s2,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-05-02,,,,
-2eae89e4-b35e-4acc-b215-c1ff4355a583,2017-07-01 00:00:00,a11,s3,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-361f52c4-b2dd-4bc6-9373-e14585c350da,2017-07-01 00:00:00,a20,s4,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-37fb369e-64be-4923-8e5f-b07ddd09f4e5,2017-07-01 00:00:00,a33,s1,,,st1,,,,,,,,,,,,,,,,,,,0,1,2017-03-02,,,,
-38378afd-4c32-4b3b-ac0b-e99ecc6e3ea9,2017-07-01 00:
```
Comment (emord) [inline custom/icds_reports/tests/fixtures/person_cases.csv]: Similarly could you add a description for what this commit did? I think that  "fix test cases" means:

1) Add dob to person cases that did not have one
2) Add child_health's person cases that did not exist before?

Is there anything else you needed to do for this file?

**Review by emord — APPROVED** [review]
The change looks reasonable. Have some questions about the test changes.

File: custom/icds_reports/tests/fixtures/child_cases.csv (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/tests/fixtures/child_cases.csv]: hey Yes, Initially these were pointing to same person case which is not a valid state.
Yes, Sure I will be putting this explanation on the commit  from next time.

File: custom/icds_reports/tests/fixtures/person_cases.csv (hunk 2 above)
Comment (Rohit25negi) [reply custom/icds_reports/tests/fixtures/person_cases.csv]: yes, This commit did what you mentioned. Sory for not putting it into the commit message

**Pull request merged**